test/e2e/fixtures/kustom/21_ssh_key_generation/generation_patch.yaml

```diff
@@ -0,0 +1,3 @@
+- op: add
+  path: /spec/template/spec/containers/0/args/-
+  value: --k8s-secret-name=flux-git-deploy-generated
```

Comment:
Why is this necessary? I thought the key was generated out of the  out of the box with what's supplied by `fluxctl install`

Reply:
Ah, you are doing this to override the key we load in the tests.

Isn't it cleaner to avoid installing that secret?

Reply:
They were left-overs from a previous version of this, I removed the arguments selecting the folder but forgot to remove the files.

Reply:
But, without doing letting flux generate the we won't be testing the big being fixed, I think.

Reply:
Without deploying the git server an empty secret will be applied and Flux will generate the key.

Reply:
Ah, good point. Can you add a comment?